### DISCUSSION TITLE
How to fix the compile error "Cannot find bean with qualifier 'metadbMigrate' "?

### DISCUSSION THREAD

**Discussion by xiaodp** [oceanbase/odc#336] [Q&A] [answered]
I am starting odc server from IDEA following the FEVELOPER_GUIDE.md and have the start parameters set.

but there reports' Cannot find bean with qualifier 'metadbMigrate' ' in src/main/java/com/oceanbase/odc/server/OdcServer.java, 
<img width="831" alt="image" src="https://github.com/oceanbase/odc/assets/24877747/d8d6d0e3-ffa8-4b69-a65c-b2efdc7f1cc6">

I am a newer to java, would any body be kind to help me with this?
the maven version: 3.9.2 
java version: 1.8.0_372

**Comment by yhilmare** [comment]
can you give us the odc.log?

**Comment by yhilmare** [comment, marked as answer]
ODC's running reply on a metaDB and we only support OceanBase as the metaDB. 

Have you correctly configured the connection information for the metadata database? If it has been configured correctly, you need to confirm whether the metadata database is running normally.

here is an example for idea to startup odc:
![截屏2023-11-03 11 59 41](https://github.com/oceanbase/odc/assets/22992296/580b912e-bc33-429f-a376-a8b444181819)

**Comment by yhilmare** [comment]
this pr may help you #850
